.github/workflows/main.yml

```diff
@@ -44,6 +44,16 @@ jobs:
             - name: Check
               run: ./gradlew check
 
-            # Gradle integration test. Includes internal docker-compose management
-            - name: Integration test
-              run: ./gradlew integrationTest
+            - name: Comment Output on Failure
+              uses: actions/github-script@v3
+              if: github.event_name == 'pull_request' && failure()
```

Comment:
Is it not enough to simply fail the test? You should already get an email from GitHub if the action fails.

Reply:
Yes, should be fine I think. I will remove this.